src/main/java/org/fundacionjala/trello/context/Context.java

```diff
@@ -1,14 +1,16 @@
 package org.fundacionjala.trello.context;
 
 import io.restassured.response.Response;
+import org.fundacionjala.trello.pages.trello.TrelloPage;
 
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
-public class Context {
+public final class Context {
 
+    private TrelloPage trelloPage;
```

Comment:
keep in mind this Context should be more generic

Reply:
Ok, It's done